setup/src/common/scripts/onCreateCommand.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) 2022-2024 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+sudo chmod +x .devcontainer/scripts/*.sh
+sudo chown -R $(whoami) $HOME
```

Comment:
Why is/was that needed?
```suggestion
```
It seems unnecessary changing the owner of the current user's "HOME tree" to become the current user

Reply:
There was some devcontainer issue which required us to do this. I am unsure whether the issue has been fixed in the meantime. Would rather not include another change in this already huge PR

Reply:
Fine for me. In the Lattice template we removed that last week - seems to work